config/default/testRunner.conf.php

```diff
@@ -575,4 +575,12 @@
      * Specifies runner tools that should keep its states in backend storage
      */
     'tool-state-server-storage' => [],
+
+    /**
+     * Defines whether to always show a warning dialog before moving to next item
+     * Also, if shown, should the dialog contain a "don't show again" checkbox
+     */
+    'force-enable-next-item-warning' => false,
+    'enable-next-item-warning-checkbox' => true,
```

Comment:
It looks obvious for us what this option is supposed to do, at least add a comment describing what the checkbox is supposed to do and this option makes sense only when the previous is activated